bridge-history-api/internal/logic/l1_event_parser.go

```diff
@@ -271,10 +271,8 @@ func (e *L1EventParser) ParseL1BatchEventLogs(ctx context.Context, logs []types.
 				return nil, err
 			}
 			l1BatchEvents = append(l1BatchEvents, &orm.BatchEvent{
-				BatchStatus:   int(btypes.BatchStatusTypeFinalized),
-				BatchIndex:    event.BatchIndex.Uint64(),
-				BatchHash:     event.BatchHash.String(),
-				L1BlockNumber: vlog.BlockNumber,
```

Comment:
still update this field, because `GetBatchEventSyncedHeightInDB` will use it as a sync height when restarting.

Reply:
this is done, and the `l1_block_bumber` field is added to updateFileds too